data/lib/core/position.lua

```diff
@@ -18,29 +18,29 @@ function Position:getNextPosition(direction, steps)
 	end
 end
 
-function Position:moveUpstairs()
-	local isWalkable = function (position)
-		local tile = Tile(position)
-		if not tile then
-			return false
-		end
+function Position:isWalkable()
+	local tile = Tile(position)
```

Comment:
You can probbly do changes here: https://github.com/Riverlance/forgottenserver/tree/patch-1
And they should be updated here when you push things there...

Option 2 install the git programs and hope for the best (that is what I do everytime, works about 70% of the times haha)

Option 3, below the comment thingi you should have an option to close the PR.